compiler/passes/normalize.cpp

```diff
@@ -866,6 +872,48 @@ static Symbol* theDefinedSymbol(BaseAST* ast) {
   return retval;
 }
 
+static bool replaceWithStridesField(Symbol* stridesField, Expr* use) {
+  SET_LINENO(use);
+  NamedExpr* ne = toNamedExpr(use->parentExpr);
+  if (ne != nullptr && !strcmp(ne->name, "stridable"))
+    ne->replace(new NamedExpr("strides", new SymExpr(stridesField)));
+  else
+    use->replace(new SymExpr(stridesField));
+  return true;
+}
+
+static bool replaceWithStridableCall(Symbol* stridesField, Expr* use) {
+  SET_LINENO(use);
+  use->replace(new CallExpr("chpl_stridable", stridesField));
+  return true;
+}
+
+// Supports deprecation by Vass in 1.31 to implement #17131.
+// chpl__buildDomainRuntimeType(..., stridable) -->
+// chpl__buildDomainRuntimeType(..., strides) if we are in a DSI class.
+bool replacedStridable(CallExpr* parentCall, const char* name,
```

Comment:
Calling this function `replacedStridable` obfuscates the fact that it has a side effect (described in the comment above).

Reply:
Renamed to `tryReplaceStridable`.